### PR TITLE
Use generic DMIC support for apl-pcm512x

### PR DESCRIPTION
**Pull request by lyakh**

### CODE REVIEW

File: tools/topology/sof-apl-pcm512x.m4 [hunk 1]
```diff
@@ -60,37 +59,32 @@ PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s32le.
 # 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
-	2, 1, 2, s32le,
+	2, 5, 2, s32le,
```
Comment (plbossart) [inline tools/topology/sof-apl-pcm512x.m4:62]: you need to change the comments as well, it's not 'playback pipeline 2 on PCM 5'. same for all changes below

Comment (lyakh) [reply tools/topology/sof-apl-pcm512x.m4:62]: @plbossart thanks for catching, fixed

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
Rescheduling CI due to DUT failure.

**Comment by lyakh** [comment]
SOFCI TEST

**Comment by lyakh** [comment]
the remaining CI errors seem unrelated, ready to merge?

**Comment by plbossart** [comment]
> the remaining CI errors seem unrelated, ready to merge?

yes, on GLK we need to filter out the message
kernel: [  626.693962] da7219 i2c-DLGS7219:00: SRM failed to lock

@fredoh9 can you take care of this?

Filed issue https://github.com/thesofproject/linux/issues/2380 for ICL-RVP boot issues.

**Comment by lyakh** [comment]
@lgirdwood I don't understand CI any more. It seems to be launching asynchronously to PRs or "SOF CITEST" requests

**Comment by lgirdwood** [comment]
Jenkins CI are known and unrelated issues.

**Pull request merged**